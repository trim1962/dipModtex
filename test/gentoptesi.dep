\RequireVersions{
  *{application}{TeX}     {1990/03/25 v3.x}
  *{format} {LaTeX2e}     {2014/05/01 v2.e}
  *{package}{snapshot}    {2002/03/05 v1.14}
  *{class}  {toptesi}     {2014/12/13 v5.86f}
  *{class}  {report}      {2014/09/29 v1.4h}
  *{file}   {size10.clo}  {2014/09/29 v1.4h}
  *{package}{ifxetex}     {2010/09/12 v0.6}
  *{package}{babel}       {2014/09/25 v3.9l}
  *{file}   {english.ldf} {2012/08/20 v3.3p}
  *{file}   {italian.ldf} {2014/03/29 v1.3k}
  *{package}{etoolbox}    {2011/01/03 v2.1}
  *{package}{etex}        {1998/03/26 v2.0}
  *{package}{toptesi}     {2014/12/13 v5.86f}
  *{package}{graphicx}    {2014/10/28 v1.0g}
  *{package}{keyval}      {2014/10/28 v1.15}
  *{package}{graphics}    {2014/10/28 v1.0p}
  *{package}{trig}        {1999/03/16 v1.09}
  *{file}   {graphics.cfg}{2010/04/23 v1.9}
  *{file}   {pdftex.def}  {2011/05/27 v0.06d}
  *{package}{infwarerr}   {2010/04/08 v1.3}
  *{package}{ltxcmds}     {2011/11/09 v1.22}
  *{package}{fontenc}     {0000/00/00 v0.0}
  *{file}   {t1enc.def}   {2005/09/27 v1.99g}
  *{package}{inputenc}    {2014/04/30 v1.2b}
  *{file}   {utf8.def}    {2014/09/29 v1.1m}
  *{file}   {t1enc.dfu}   {2014/09/29 v1.1m}
  *{file}   {ot1enc.dfu}  {2014/09/29 v1.1m}
  *{file}   {omsenc.dfu}  {2014/09/29 v1.1m}
  *{file}   {supp-pdf.mkii}{0000/00/00 v0.0}
  *{package}{pdftexcmds}  {2011/11/29 v0.20}
  *{package}{ifluatex}    {2010/03/01 v1.3}
  *{package}{ifpdf}       {2011/01/30 v2.3}
  *{package}{epstopdf-base}{2010/02/09 v2.5}
  *{package}{grfext}      {2010/08/19 v1.1}
  *{package}{kvdefinekeys}{2011/04/07 v1.3}
  *{package}{kvoptions}   {2011/06/30 v3.11}
  *{package}{kvsetkeys}   {2012/04/25 v1.16}
  *{package}{etexcmds}    {2011/02/16 v1.5}
  *{file}   {epstopdf-sys.cfg}{2010/07/13 v1.3}
  *{package}{topfront}    {2014/12/13 v5.86f}
  *{file}   {toptesi.cfg} {0000/00/00 v0.0}
  *{package}{topcoman}    {2014/12/13 v5.86f}
  *{package}{textcomp}    {2005/09/27 v1.99g}
  *{file}   {ts1enc.def}  {2001/06/05 v3.0e}
  *{file}   {ts1enc.dfu}  {2014/09/29 v1.1m}
  *{file}   {t1cmtt.fd}   {2014/09/29 v2.5h}
}
