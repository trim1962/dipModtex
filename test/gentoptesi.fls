PWD C:/tex/geneatex/test
INPUT c:/texlive/2014/texmf.cnf
INPUT c:/texlive/2014/texmf-dist/web2c/texmf.cnf
INPUT c:/texlive/2014/texmf-var/web2c/pdftex/pdflatex.fmt
INPUT gentoptesi.tex
OUTPUT gentoptesi.log
INPUT c:/texlive/2014/texmf-dist/tex/latex/snapshot/snapshot.sty
INPUT c:/texlive/2014/texmf-dist/tex/latex/snapshot/snapshot.sty
INPUT c:/texlive/2014/texmf-dist/tex/latex/toptesi/toptesi.cls
INPUT c:/texlive/2014/texmf-dist/tex/latex/toptesi/toptesi.cls
INPUT c:/texlive/2014/texmf-dist/tex/latex/base/report.cls
INPUT c:/texlive/2014/texmf-dist/tex/latex/base/report.cls
INPUT c:/texlive/2014/texmf-dist/tex/latex/base/size10.clo
INPUT c:/texlive/2014/texmf-dist/tex/latex/base/size10.clo
INPUT c:/texlive/2014/texmf-dist/tex/generic/ifxetex/ifxetex.sty
INPUT c:/texlive/2014/texmf-dist/tex/generic/ifxetex/ifxetex.sty
INPUT c:/texlive/2014/texmf-dist/tex/generic/babel/babel.sty
INPUT c:/texlive/2014/texmf-dist/tex/generic/babel/babel.sty
INPUT c:/texlive/2014/texmf-dist/tex/generic/babel-english/english.ldf
INPUT c:/texlive/2014/texmf-dist/tex/generic/babel-english/english.ldf
INPUT c:/texlive/2014/texmf-dist/tex/generic/babel/babel.def
INPUT c:/texlive/2014/texmf-dist/tex/generic/babel-italian/italian.ldf
INPUT c:/texlive/2014/texmf-dist/tex/generic/babel-italian/italian.ldf
INPUT c:/texlive/2014/texmf-dist/tex/latex/etoolbox/etoolbox.sty
INPUT c:/texlive/2014/texmf-dist/tex/latex/etoolbox/etoolbox.sty
INPUT c:/texlive/2014/texmf-dist/tex/latex/etex-pkg/etex.sty
INPUT c:/texlive/2014/texmf-dist/tex/latex/etex-pkg/etex.sty
INPUT c:/texlive/2014/texmf-dist/tex/latex/toptesi/toptesi.sty
INPUT c:/texlive/2014/texmf-dist/tex/latex/toptesi/toptesi.sty
INPUT c:/texlive/2014/texmf-dist/tex/latex/graphics/graphicx.sty
INPUT c:/texlive/2014/texmf-dist/tex/latex/graphics/graphicx.sty
INPUT c:/texlive/2014/texmf-dist/tex/latex/graphics/keyval.sty
INPUT c:/texlive/2014/texmf-dist/tex/latex/graphics/keyval.sty
INPUT c:/texlive/2014/texmf-dist/tex/latex/graphics/graphics.sty
INPUT c:/texlive/2014/texmf-dist/tex/latex/graphics/graphics.sty
INPUT c:/texlive/2014/texmf-dist/tex/latex/graphics/trig.sty
INPUT c:/texlive/2014/texmf-dist/tex/latex/graphics/trig.sty
INPUT c:/texlive/2014/texmf-dist/tex/latex/latexconfig/graphics.cfg
INPUT c:/texlive/2014/texmf-dist/tex/latex/latexconfig/graphics.cfg
INPUT c:/texlive/2014/texmf-dist/tex/latex/pdftex-def/pdftex.def
INPUT c:/texlive/2014/texmf-dist/tex/latex/pdftex-def/pdftex.def
INPUT c:/texlive/2014/texmf-dist/tex/generic/oberdiek/infwarerr.sty
INPUT c:/texlive/2014/texmf-dist/tex/generic/oberdiek/infwarerr.sty
INPUT c:/texlive/2014/texmf-dist/tex/generic/oberdiek/ltxcmds.sty
INPUT c:/texlive/2014/texmf-dist/tex/generic/oberdiek/ltxcmds.sty
INPUT gentoptesi.aux
INPUT gentoptesi.aux
OUTPUT gentoptesi.aux
INPUT c:/texlive/2014/texmf-dist/tex/context/base/supp-pdf.mkii
INPUT c:/texlive/2014/texmf-dist/tex/context/base/supp-pdf.mkii
INPUT c:/texlive/2014/texmf-dist/tex/generic/oberdiek/pdftexcmds.sty
INPUT c:/texlive/2014/texmf-dist/tex/generic/oberdiek/pdftexcmds.sty
INPUT c:/texlive/2014/texmf-dist/tex/generic/oberdiek/ifluatex.sty
INPUT c:/texlive/2014/texmf-dist/tex/generic/oberdiek/ifluatex.sty
INPUT c:/texlive/2014/texmf-dist/tex/generic/oberdiek/ifpdf.sty
INPUT c:/texlive/2014/texmf-dist/tex/generic/oberdiek/ifpdf.sty
INPUT c:/texlive/2014/texmf-dist/tex/latex/oberdiek/epstopdf-base.sty
INPUT c:/texlive/2014/texmf-dist/tex/latex/oberdiek/epstopdf-base.sty
INPUT c:/texlive/2014/texmf-dist/tex/latex/oberdiek/grfext.sty
INPUT c:/texlive/2014/texmf-dist/tex/latex/oberdiek/grfext.sty
INPUT c:/texlive/2014/texmf-dist/tex/generic/oberdiek/kvdefinekeys.sty
INPUT c:/texlive/2014/texmf-dist/tex/generic/oberdiek/kvdefinekeys.sty
INPUT c:/texlive/2014/texmf-dist/tex/latex/oberdiek/kvoptions.sty
INPUT c:/texlive/2014/texmf-dist/tex/latex/oberdiek/kvoptions.sty
INPUT c:/texlive/2014/texmf-dist/tex/generic/oberdiek/kvsetkeys.sty
INPUT c:/texlive/2014/texmf-dist/tex/generic/oberdiek/kvsetkeys.sty
INPUT c:/texlive/2014/texmf-dist/tex/generic/oberdiek/etexcmds.sty
INPUT c:/texlive/2014/texmf-dist/tex/generic/oberdiek/etexcmds.sty
INPUT c:/texlive/2014/texmf-dist/tex/latex/latexconfig/epstopdf-sys.cfg
INPUT c:/texlive/2014/texmf-dist/tex/latex/latexconfig/epstopdf-sys.cfg
INPUT c:/texlive/2014/texmf-dist/tex/latex/toptesi/topfront.sty
INPUT c:/texlive/2014/texmf-dist/tex/latex/toptesi/topfront.sty
INPUT c:/texlive/2014/texmf-dist/tex/latex/toptesi/toptesi.cfg
INPUT c:/texlive/2014/texmf-dist/tex/latex/toptesi/toptesi.cfg
INPUT c:/texlive/2014/texmf-dist/tex/latex/toptesi/toptesi.cfg
INPUT c:/texlive/2014/texmf-dist/tex/latex/toptesi/topcoman.sty
INPUT c:/texlive/2014/texmf-dist/tex/latex/toptesi/topcoman.sty
INPUT c:/texlive/2014/texmf-dist/tex/latex/base/textcomp.sty
INPUT c:/texlive/2014/texmf-dist/tex/latex/base/textcomp.sty
INPUT c:/texlive/2014/texmf-dist/tex/latex/base/ts1enc.def
INPUT c:/texlive/2014/texmf-dist/tex/latex/base/ts1enc.def
INPUT c:/texlive/2014/texmf-dist/fonts/map/fontname/texfonts.map
INPUT c:/texlive/2014/texmf-dist/fonts/tfm/public/cm/cmtt10.tfm
INPUT c:/texlive/2014/texmf-dist/fonts/tfm/public/cm/cmtt8.tfm
INPUT c:/texlive/2014/texmf-dist/fonts/tfm/public/cm/cmcsc10.tfm
OUTPUT gentoptesi.pdf
INPUT c:/texlive/2014/texmf-var/fonts/map/pdftex/updmap/pdftex.map
INPUT gentoptesi.aux
OUTPUT gentoptesi.dep
INPUT c:/texlive/2014/texmf-dist/fonts/type1/public/amsfonts/cm/cmcsc10.pfb
INPUT c:/texlive/2014/texmf-dist/fonts/type1/public/amsfonts/cm/cmr10.pfb
