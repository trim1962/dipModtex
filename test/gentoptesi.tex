\RequirePackage{snapshot}
\documentclass{toptesi}
\listfiles
\begin{document}
	prova
\end{document}
