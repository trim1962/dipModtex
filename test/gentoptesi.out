\BOOKMARK [0][]{chapter*.1}{Sommario}{}% 1
\BOOKMARK [0][]{chapter*.4}{Elenco delle tabelle}{}% 2
\BOOKMARK [0][]{chapter*.5}{Elenco delle figure}{}% 3
\BOOKMARK [-1][]{part.1}{I Prima Parte}{}% 4
\BOOKMARK [0][]{chapter.1}{Introduzione generale}{part.1}% 5
\BOOKMARK [1][]{section.1.1}{Principi generali}{chapter.1}% 6
\BOOKMARK [1][]{section.1.2}{I satelliti medicei}{chapter.1}% 7
\BOOKMARK [0][]{chapter.2}{Il barometro}{part.1}% 8
\BOOKMARK [1][]{section.2.1}{Generalit\340}{chapter.2}% 9
\BOOKMARK [2][]{subsection.2.1.1}{Forma del barometro}{section.2.1}% 10
\BOOKMARK [1][]{section.2.2}{Del mercurio}{chapter.2}% 11
\BOOKMARK [0][]{chapter.3}{Il listato del pacchetto topcoman.sty}{part.1}% 12
\BOOKMARK [0][]{chapter*.6}{Bibliografia}{part.1}% 13
